### DISCUSSION TITLE
Support for width/height props for Dialog incubator

### DISCUSSION THREAD

**Discussion by MuhammedKpln** [wix/react-native-ui-lib#1795] [Ideas]
Old dialog component, you can pass width and height, but with the new component we cannot pass it and as a result it has a huge padding on sides as you can see below.

<img width="314" alt="Screenshot 2022-01-21 at 10 53 39" src="https://user-images.githubusercontent.com/17166855/150506020-457060de-6b70-47c6-82c2-11a19b0017d4.png">

**Comment by ethanshar** [comment]
Maybe @M-i-k-e-l can assist here..

**Comment by M-i-k-e-l** [comment]
Hi @MuhammedKpln,
If you want to override the default style you can send `containerStyle` with your own style.
If you mean sending the modifiers `width` \ `height`, then I've added it to our TODOs.
